flume-ng-sinks/flume-ng-kafka-sink/src/main/java/org/apache/flume/sink/kafka/KafkaSink.java

```diff
@@ -421,7 +424,12 @@ private void setProducerProps(Context context, String bootStrapServers) {
     kafkaProps.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, DEFAULT_VALUE_SERIAIZER);
     kafkaProps.putAll(context.getSubProperties(KAFKA_PRODUCER_PREFIX));
     kafkaProps.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, bootStrapServers);
-
+    //  The default value of `ssl.endpoint.identification.algorithm` is changed to `https`, since kafka client 2.0+
+    //  And because flume does not accept an empty string as property value,
+    //  so we need to use alternative custom property `ssl.disableTLSHostnameVerification` to check if enable fqdn check.
```

Comment:
This line is failing checkstyle because it is one character too long

Reply:
fixed